.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 .metadata/
+.idea/
```

Comment:
I added this as a convenience for anyone opening the repository root in Android Studio.